edge-util/src/Microsoft.Azure.Devices.Edge.Storage.RocksDb/ColumnFamilyDbStore.cs

```diff
@@ -54,13 +63,16 @@ public Task Put(byte[] key, byte[] value, CancellationToken cancellationToken)
             Preconditions.CheckNotNull(key, nameof(key));
             Preconditions.CheckNotNull(value, nameof(value));
 
+            this.count += 1;
```

Comment:
might not be super important, but in case op gets cancelled, the counter will be off

```csharp
            await operation.ExecuteUntilCancelled(cancellationToken);
            this.count++;
```

Reply:
Good point! Will update quick

Reply:
Hmm... I can do something slightly different to your suggestion as I just noticed the method is not async so await cannot be used... but I can make the count changes happen after the db.put and db.remove :)

Reply:
@nyanzebra, I noticed the same thing, but later realized that action is executed in a separate task, so you need sync access to the counter :) (`Interlocked`). I don't see a reason why you can't make the method async?

Reply:
I am okay with doing that, and good point on the counter, didn't think of that, one moment